shared/hashutil/hash.go

```diff
@@ -1,15 +1,16 @@
 package hashutil
 
 import (
-	"golang.org/x/crypto/blake2b"
+	"golang.org/x/crypto/sha3"
 )
 
 // Hash defines a function that returns the
-// blake2b hash of the data passed in.
+// Keccak-256/SHA3 hash of the data passed in.
 // https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#appendix
 func Hash(data []byte) [32]byte {
 	var hash [32]byte
-	h := blake2b.Sum512(data)
+
+	h := sha3.Sum256(data)
```

Comment:
which sha3 is this ? the keccak-256 used by ethereum is different then what is finalised as sha3 as a standard 
For reference: 
https://ethereum.stackexchange.com/questions/550/which-cryptographic-hash-function-does-ethereum-use

Reply:
Yes this is different @terenc3t - I think we should just use go-ethereum/common with its BytesToHash methods everywhere instead and deprecate our hashutils package